Source/WebGPU/WebGPU/TextureView.mm

```diff
@@ -51,6 +53,16 @@
     m_texture.label = label;
 }
 
+void TextureView::recreateBackingWithStorageIfNeeded()
+{
+    if (m_texture.storageMode != MTLStorageModeMemoryless)
+        return;
+
+    ASSERT(m_textureBacking);
+    m_textureBacking->recreateBackingWithStorage();
+    m_texture = m_textureBacking->texture();
```

Comment:
In order to support memoryless textures, `TextureView::m_texture` should point to the same object as `Texture::m_texture` and this is what this patch does. 

We don't need to worry about bind groups holding references to memoryless textures because memoryless is only created when `descriptor.usage == WGPUTextureUsage_RenderAttachment` and to be in a bind group, the resource needs `descriptor.usage | WGPUTextureUsage_TextureBinding` 

There is an issue as you point out with multiple render target TextureView instances referencing the same texture. 

If you create multiple TextureView instances from the same render target, some with storeOp=store and some with storeOp=discard, then some of the TextureView instances with reference the memoryless texture and others will reference the memory-backed texture. That seems wrong since the backing Texture instance will no longer have a reference to the memoryless texture.

I'm not sure if it would cause incorrect behavior but I'll fix that issue because it seems at least it would be confusing